Source/dom/models/__tests__/CurvePoint.test.js

```diff
@@ -47,6 +47,6 @@ test('should show if a point is selected', () => {
     .eventHandlerManager()
     .switchToEventHandlerClass(MSShapeEventHandler.class())
 
-  expect(shape.points[0].isSelected()).toBe(true)
+  expect(shape.points[0].isSelected()).toBe(false)
```

Comment:
After a chat with @christianklotz 🙏, finally found the motivation to fire VS Code and write the following instead of writing dumb js.

```suggestion
  // ask handler to select any point; by default, nothing's selected.
  document.sketchObject
    .eventHandlerManager()
    .currentHandler()
    .pathController()
    .selectNext(null)

  expect(shape.points[0].isSelected()).toBe(true)
```

Reply:
third time is the charm.  this is what I get for trying to do this in a hurry before a meeting.